ETC_en.tsv

```diff
@@ -5164,24 +5164,24 @@ ETC_20150317_005163	Please defeat Moyabruka!
 ETC_20150317_005164	Polluted Pond 1
 ETC_20150317_005165	The relax bomb slows down your movement {nl} when you approach Upent!
 ETC_20150317_005166	So you are still alive...
-ETC_20150317_005167	Your role is done.{nl}Die in silence!
+ETC_20150317_005167	You're done.{nl}Die in silence!
 ETC_20150317_005168	Goddess Shaule
 ETC_20150317_005169	Restraining Magic Circle
 ETC_20150317_005170	Temple Border
 ETC_20150317_005171	Goddess Shaule
 ETC_20150317_005172	Portal
-ETC_20150317_005173	Please defeat Golem!
+ETC_20150317_005173	Please defeat the Golem!
 ETC_20150317_005174	Please defeat Spanashon!
 ETC_20150317_005175	Please defeat Yekub!
 ETC_20150317_005176	Patient
 ETC_20150317_005177	Trigger to kill soldiers
-ETC_20150317_005178	Please defeat Yonazolem and obtain the easy burning stone!
+ETC_20150317_005178	Defeat Yonazolem and obtain the faintly burning stone!
 ETC_20150317_005179	Let the fight begin!
 ETC_20150317_005180	Mines Manager
-ETC_20150317_005181	See? No just a day or two...
+ETC_20150317_005181	See? Not just a day or two...
 ETC_20150317_005182	Argh! There it is!!!
-ETC_20150317_005183	Please defeat Shadowgaler
-ETC_20150317_005184	Please defeat Minotaur!
+ETC_20150317_005183	Please defeat the Shadowgaler
+ETC_20150317_005184	Please defeat the Minotaur!
```

Comment:
Seems to be boss names : 
ITEM_20150317_002003	Minotaur Card
ITEM_20150317_002015	Shadowgaler Card